test/test_layout.cpp

```diff
@@ -16,68 +16,78 @@
 
 #include "doctest/doctest.h"
 
-#include "sparrow/layout.hpp"
+#include "sparrow/fixed_size_layout.hpp"
 
 namespace sparrow
 {
     // TODO: Test all the other base types once #15 is addressed.
-    static_assert(std::ranges::contiguous_range<primitive_layout<uint8_t>>);
-    static_assert(std::contiguous_iterator<primitive_layout_iterator<uint8_t, true>>);
-    static_assert(std::contiguous_iterator<primitive_layout_iterator<uint8_t, false>>);
+    // static_assert(std::ranges::contiguous_range<fixed_size_layout<uint8_t>>);
```

Comment:
It cannot be a range because it doesnt expose `iterator` (which should be what `value_iterator` is) and no begin/end directly (which should be just what `value_range()` exposes). We can do that, it would make sense IMO.

Reply:
When https://github.com/xtensor-stack/sparrow/pull/41 is merged, we should be able to adapt those assertions, yes.